src/api/operator/types.ts

```diff
@@ -20,31 +22,14 @@ export interface FeeInformation {
 
 export type OrderKind = 'sell' | 'buy'
 
+export type SigningSchemeValue = 'eip712' | 'ethsign' | 'eip1271' | 'presign'
 export type OrderStatus = 'open' | 'filled' | 'cancelled' | 'cancelling' | 'expired' | 'signing'
 export type RawOrderStatusFromAPI = 'presignaturePending' | 'open' | 'fullfilled' | 'cancelled' | 'expired'
 
 // Raw API response
-export type RawOrder = {
-  creationDate: string
-  owner: string
-  receiver?: string
-  uid: string
-  executedBuyAmount: string
-  executedSellAmount: string
-  executedFeeAmount: string
-  invalidated: boolean
-  sellToken: string
-  buyToken: string
-  sellAmount: string
-  buyAmount: string
-  validTo: number
-  appData: number
-  feeAmount: string
-  kind: OrderKind
-  partiallyFillable: boolean
-  signature: string
-  status: RawOrderStatusFromAPI
-}
+export type RawOrder =
+  | (OrderMetaData & { executedSellAmountBeforeFees: string; signingScheme: SigningSchemeValue })
+  | (Omit<OrderMetaData, 'partiallyFillable' | 'invalidated'> & { partiallyFillable: boolean; invalidated: boolean })
```

Comment:
I still think this does not need to be different than what the API returns.
If the RawOrder here has more fields than the SDK provides, they should be added to the SDK as it should match what the backend offers
If RawOrder on the other hand has more fields than the API, it's not so raw after all and should we should move those fields to the inner type.

Or did I get this wrong?